Library/Formula/hugo.rb

```diff
@@ -105,10 +105,19 @@ class Hugo < Formula
     url "https://github.com/yosssi/ace.git",
         :revision => "1f82044938a7180f6fb6bbb3a29688d1e6dbe74b"
   end
-  go_resource "gopkg.in/fsnotify.v0" do
-    url "https://github.com/go-fsnotify/fsnotify.git",
+  stable do
+    # remove in next release
```

Comment:
Can you clarify a bit in the comment why this is needed and what's going on here? Thanks!